packs/data/extinction-curse-bestiary.db/counteflora.json

```diff
@@ -292,7 +378,7 @@
                     "value": 2
                 },
                 "description": {
-                    "value": "<p>The counteflora releases a cloud of black seedpods in a 15-foot-radius emanation. Non-counteflora creatures in the area must succeed at a Fortitude save or breathe in the toxic seeds.</p>"
+                    "value": "<p><strong>Requirements:</strong> the Counteflora's bracts are not tightened.</p>\n<hr />\n<p>The counteflora releases a cloud of black seedpods in a @Template[type:emanation|distance:15]{15-foot-radius emanation}. Non-counteflora creatures in the area must succeed at a Fortitude save or breathe in the toxic seeds. (Against plants and fungi, the seeds burrow into the creatures' flesh rather than infiltrate their lungs.)</p>"
```

Comment:
```suggestion
                    "value": "<p><strong>Requirements</strong> the Counteflora's bracts are not tightened.</p>\n<hr />\n<p>The counteflora releases a cloud of black seedpods in a @Template[type:emanation|distance:15]{15-foot-radius emanation}. Non-counteflora creatures in the area must succeed at a Fortitude save or breathe in the toxic seeds. (Against plants and fungi, the seeds burrow into the creatures' flesh rather than infiltrate their lungs.)</p>"
```